easybuild/easyconfigs/p/preCICE/preCICE-1.3.0_fix_bad_symlinking.patch

```diff
@@ -0,0 +1,14 @@
+diff -ru precice-1.3.0.orig/CMakeLists.txt precice-1.3.0/CMakeLists.txt
```

Comment:
@akesandgren missing patch description + author

Did you report this upstream?

Reply:
he he... the checksum is correct, I just forgot to upload the patchfile after fixing the checksums for the updated patchfile.

Reply:
Greetings from upstream! We finally removed the unfortunate symlink contraption after the 1.3.0 release. https://github.com/precice/precice/commit/0f6b0a836dc3212cfb8806fe20f06c6ad8c265c7

@akesandgren many thanks for this PR! FYI we are planning to release 1.4.0 in around 2 weeks.

Reply:
@fsimonis :+1: :)